modules/core/src/smithy4s/syntax/package.scala

```diff
@@ -45,9 +31,53 @@ package object syntax
     with schematic.suspended.ClosedSyntax[Schematic]
     with schematic.bijection.ClosedSyntax[Schematic]
     with Hints.ClosedSyntax[Schematic]
-    with Document.ClosedSyntax[Schematic]
     with errorUnion.Syntax {
 
+  val short: Schema[Short] =
```

Comment:
the changes bellow should address the concern of not having shape ids for primitive types